lib/ansible/modules/network/ldap_entry.py

```diff
@@ -268,17 +287,18 @@ def main():
             'server_uri': dict(default='ldapi:///'),
             'start_tls': dict(default=False, type='bool'),
             'state': dict(default='present', choices=['present', 'absent']),
+            'validate_certs': dict(default=True, type='bool'),
```

Comment:
Actually, the option `validate_certs` is also used in the `get_url` module. I think we should keep it in sync with other modules.